packages/http/src/mocker/HttpMocker.ts

```diff
@@ -46,7 +46,7 @@ export class HttpMocker implements IMocker<IHttpOperation, IHttpRequest, IHttpCo
     let negotiationResult: IHttpNegotiationResult;
     if (input.validations.input.length > 0) {
       try {
-        negotiationResult = helpers.negotiateOptionsForInvalidRequest(resource.responses);
+        negotiationResult = helpers.negotiateOptionsForInvalidRequest(resource.responses!);
```

Comment:
`resource.responses!`
Is this an oversight?

Reply:
Yes! 88bb828